crates/rpc/rpc-eth-api/src/core.rs

```diff
@@ -810,4 +818,14 @@ where
         trace!(target: "rpc::eth", ?address, ?keys, ?block_number, "Serving eth_getProof");
         Ok(EthState::get_proof(self, address, keys, block_number)?.await?)
     }
+
+    /// Handler for: `eth_getAccountInfo`
+    async fn get_account_info(
+        &self,
+        address: Address,
+        block: BlockId,
+    ) -> RpcResult<Option<alloy_rpc_types_eth::AccountInfo>> {
+        trace!(target: "rpc::eth", "Serving eth_getAccountInfo");
+        Ok(self.get_account_info(address, block).await?)
```

Comment:
this looks promising, what we need to do is add another `EthState` function that does this and looks up the account